### ISSUE TITLE
Lagi na stronie cdi.multilada.pl

### ISSUE THREAD

**Issue by gkrajewski** [pikrol/CDI-static#40]
Zauważyłem, że czasem apka się wiesza na kilka długich sekund, jak klikam "dalej" albo "zatwierdź" (a może tylko przy jednym z tych przycisków... nie pamiętam). Nie wiem, czy da się coś z tym zrobić, ale to jest potencjalnie problematyczne.

**Comment by pikrol** [comment]
Jedyne wieszanie jakie udało mi się znaleźć to przy kliknięciu "Zakończ". To jest chyba związane z tym, że wysyła się ten mail z wynikami. Mogę tam ewentualnie dołożyć jakieś "Proszę czekać".

Jakby takie rzeczy się jeszcze zdarzały to proszę wyłapać dokładnie w którym momencie, bo inaczej to sobie mogę klikać i szukać :)

**Comment by gkrajewski** [comment]
Oczywiście, będę wyłapywał! Dopiszę to też zaraz do pliku Karoliny ze ścieżką krytyczną.
